crates/aptos-protos/proto/aptos/transaction/v1/transaction.proto

```diff
@@ -403,7 +403,7 @@ message Signature {
     TYPE_MULTI_ED25519 = 2;
     TYPE_MULTI_AGENT = 3;
     TYPE_FEE_PAYER = 4;
-    TYPE_SECP256K1_ECDSA = 5;
+    TYPE_SINGLE_SENDER = 6;
```

Comment:
We might not want to remove fields instead we can do `reserved 5` in this case.

Reply:
This never made it to mainnet or testnet. If you really want that though I can update.